docs(aurora): fix permission scopes (#921)
armory/docs: content/en/docs/installation/aurora-install.md

## Create client credentials for your Agents

> Use client credentials for authentication and authorization for Armory's hosted cloud services. For example, RNAs use these credentials to authenticate itself to Armory's hosted cloud services.

Reviewer: ```suggestion
Use client credentials for authentication and authorization for Armory's hosted cloud services. Services such as RNAs use these credentials to authenticate to Armory's hosted cloud services.
```

1. Log in to the Armory Cloud Console: https://console.cloud.armory.io/.
2. If you have more than one environment, ensure the proper environment is selected in the user context menu:

   {{< figure src="/images/deploy-engine/cloud-env-context.png" alt="The upper right section of the window shows what environment you are currently in." >}}

3. In the left navigation menu, select **Access Management > Client Credentials**.
4. In the upper right corner, select **New Credential**.

> 5. Create a credential for your RNAs. Use a descriptive name for the credential, such as `RNA for TV App K8s`

Reviewer: ```suggestion
5. Create a credential for your RNA. Use a descriptive name for the credential, such as `RNA for TV App K8s`
```
